tests/browser/http/test_content_disposition.py

```diff
@@ -140,53 +144,55 @@ def test_attonly(self):
         reply = stubs.FakeNetworkReply(
             headers={'Content-Disposition': 'attachment'})
         cd_inline, cd_filename = http.parse_content_disposition(reply)
-        self.assertFalse(cd_inline)
-        self.assertEqual(cd_filename, DEFAULT_NAME)
+        assert not cd_inline
+        assert cd_filename == DEFAULT_NAME
 
-    def test_attonlyquoted(self):
+    def test_attonlyquoted(self, header_checker):
         """'attachment' only, using double quotes
 
         This is invalid syntax, thus the header should be ignored.
         """
-        self._check_ignored('"attachment"')
+        header_checker.check_ignored('"attachment"')
 
     # we can't test attonly403 here.
 
-    def test_attonlyucase(self):
+    def test_attonlyucase(self, header_checker):
```

Comment:
We should create an additional PR that renames these tests as they are pretty difficult to understand for someone who is new to the project. :worried: